zazu/repo/commands.py

```diff
@@ -51,10 +55,22 @@ def ci(ctx):
 
 
 @repo.command()
+@click.argument('repository')
```

Comment:
The name of this should tell you more about what the expected format is. Is this a URL? maybe call it `repository_url`